### PR TITLE
Release notes for 2.19.0.

### PR DESCRIPTION
**Pull request by BorisDog**

### CODE REVIEW

**Review by DmitryLukyanov — APPROVED** [review]
LGTM

File: Release Notes/Release Notes v2.19.0.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+What types are considered safe is determined by a new configurable `AllowedTypes` function (of type `Func<Type, bool>`).
+The default `AllowedTypes` function is `ObjectSerializer.DefaultAllowedTypes` which returns true for a number of well-known framework types that we have deemed safe.
+A typical example might be to allow all the default allowed types as well as your own types. This could be accomplished as follows:
+
+```
+var objectSerializer = new ObjectSerializer(type => ObjectSerializer.DefaultAllowedTypes(type) || type.FullName.StartsWith("MyNamespace"));
+BsonSerializer.RegisterSerializer(objectSerializer);
+```
+
+More information about the ObjectSerializer is available at FAQ page at:
```
Comment (JamesKovacs) [inline Release Notes/Release Notes v2.19.0.md:27]: ObjectSerializer => `ObjectSerializer`
(e.g. backticks around the class name)

>is available at FAQ page at:

is available in [our FAQ](https://www.mongodb.com/docs/drivers/csharp/v2.19/faq).

The FAQ link should be correct based on the docs staging site.

**Review by JamesKovacs — CHANGES_REQUESTED** [review]
Minor changes.

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Pull request merged**